index.js

```diff
@@ -1,9 +1,21 @@
 const express = require('express');
 const app = express();
 const PORT = 4000;
+const sessions = require('./controllers/sessions')
```

Comment:
put the import (`const sessions = require('./controllers/sessions')`) at the top of the file, it is a convention